LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2015–17 Hack Cambridge
```

Comment:
Ah — I took the first commit date instead.  Out of interest, where did you find the comma-separated list requirement?  Almost all of the open source repositories I’ve seen use a range.